persistance/src/main/scala/model/User.scala

```diff
@@ -0,0 +1,58 @@
+package model
+
+import com.datastax.driver.core.ResultSet
+import com.outworkers.phantom.CassandraTable
+import com.outworkers.phantom.connectors.CassandraConnection
+import com.outworkers.phantom.dsl._
+
+import scala.concurrent.Future
+
+case class UserInfo(email: String, passWord: String, category: Category.Value)
```

Comment:
@AyushTiwary  case class UserInfo Should be in common i am using the same in notification-api and user-api and i think category should be optional by default it will be trainee

Reply:
@anubhav100 I changed the category to be optional but it quite strange that the value that is always present in our store i.e. Cassandra marked it as optional